src/Pim/Component/Catalog/Updater/Copier/MetricAttributeCopier.php

```diff
@@ -96,8 +96,8 @@ protected function copySingleValue(
         if (null !== $fromValue) {
             $fromData = $fromValue->getData();
             $toValue = $toProduct->getValue($toAttribute->getCode(), $toLocale, $toScope);
-            if (null === $toValue) {
-                $toValue = $this->productBuilder->addProductValue($toProduct, $toAttribute, $toLocale, $toScope);
+            if (null !== $toValue) {
```

Comment:
could be cool to have this logic of value removal in the addProductValue directly
because that's always what we do